packages/orchestration/src/utils/amounts.js

```diff
@@ -52,3 +52,29 @@ export const coerceCoin = (chainHub, amount) => {
     amount: String(amount.value),
   });
 };
+
+/**
+ * @param {ChainHub} chainHub
+ * @param {Denom} denom
+ * @returns {DenomDetail | undefined}
+ */
+export const getAssetSafe = (chainHub, denom) => {
+  try {
+    return chainHub.getAsset(denom);
+  } catch (_) {
+    return undefined;
+  }
+};
+
+/**
+ * @param {ChainHub} chainHub
+ * @param {Brand<'nat'>} brand
+ * @returns {Denom | undefined}
+ */
+export const getDenomSafe = (chainHub, brand) => {
+  try {
+    return chainHub.getDenom(brand);
+  } catch (_) {
+    return undefined;
+  }
+};
```

Comment:
In the course of adding these, I considered refactoring these helpers to something like: 

```ts
type DenomAmountHelper = {
  coerceDenom: (denomArg: DenomArg) => Denom;
  coerceDenomAmount: (amount: DenomAmount | Amount<'nat'>) => DenomAmount;
  coerceCoin: (amount: AmountArg | Amount<'nat'>) => Coin;
  getAssetSafe: (denom: Denom) => DenomDetail | undefined;
  getDenomSafe: (brand: Brand<'nat'>) => Denom | undefined;
};

type makeDenomAmountHelper = (chainHub: ChainHub) => DenomAmountHelper;
````

so we can use it more conveniently.  However, that might beg the question as to why these helpers  aren't in ChainHub.

Anyone have thoughts/opinions here? cc @turadg @dckc

Reply:
I think we should drop the Safe methods. I don't have a strong opinion on whether to curry the chainhub into the coercers but I lean towards the current factoring as it's more flexible.